src/views/Project/Details/Users/ProjectUsergroupList/Actions/index.js

```diff
@@ -148,6 +168,10 @@ export default class Actions extends React.PureComponent {
             },
         );
 
+        const isSuperior = this.getUserRoleLevel(projectRoleList, role) < activeUserRole.level;
+        const filteredProjectRoleList = isSuperior ?
```

Comment:
Why not filter anyway?

Reply:
Missed its use.
It is filtered to not show roles above the users role and also disable for superior roles.

Reply:
why not just do:
```
const filteredProjectRoleList = this.filterProjectRole(projectRoleList, activeUserRole.level);
```